cmd/logs.go

```diff
@@ -27,31 +27,43 @@ import (
 
 // logsCmd represents the logs command
 var logsCmd = &cobra.Command{
-	Use:   "logs",
+	Use:   "logs <stack_name>",
```

Comment:
Should probably be consistent about `[positional_arg]` or `<positional_arg>` in these descriptions

Reply:
Oh I guess you're trying to show optional vs. required args?

Reply:
Maybe use Unix style `ARG` vs `[ARG]`...

Reply:
I think the convention is normally `<required_arg>` or `[optional_arg]`